packages/errors/src/messages.ts

```diff
@@ -501,7 +501,8 @@ export const SolanaErrorMessages: Readonly<{
     [SOLANA_ERROR__SUBTLE_CRYPTO__GENERATE_FUNCTION_UNIMPLEMENTED]: 'No key generation implementation could be found.',
     [SOLANA_ERROR__SUBTLE_CRYPTO__SIGN_FUNCTION_UNIMPLEMENTED]: 'No signing implementation could be found.',
     [SOLANA_ERROR__SUBTLE_CRYPTO__VERIFY_FUNCTION_UNIMPLEMENTED]: 'No key export implementation could be found.',
-    [SOLANA_ERROR__TIMESTAMP_OUT_OF_RANGE]: 'Timestamp value must be in the range [-8.64e15, 8.64e15]. `$value` given',
```

Comment:
This range is because of this: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date#the_epoch_timestamps_and_invalid_date

Reply:
But the RPC isn't returning a value that's guaranteed to fit in a JavaScript `Date`.

Reply:
Guh.

Is there a point to the unix timestamp assertion methods then?

Reply:
We like consistent APIs? 😅

In all seriousness, this may seem like a silly change, but it's _technically_ the spec.

Reply:
We could add to the API for `UnixTimestamp` for compatibility with JavaScript's `Date`. Something like:

```ts
function isValidJavaScriptDate(unixTimestamp: UnixTimestamp): bool
```

Reply:
Only if that's an assertion function; otherwise you might as well just try/catch it.

```ts
function isValidJavaScriptDate(unixTimestamp: UnixTimestamp): unixTimestamp is UnixTimestampCompatibleWithDateConstructor { ... }
```

…or whatever.